packages/axelar-wasm-std/src/utils.rs

```diff
@@ -19,6 +22,22 @@ impl<T> TryMapExt<T> for Vec<T> {
     }
 }
 
+/// since the `Felt` type doesn't error on overflow, we have to implement that check
```

Comment:
I was also contemplating creating something like a `FeltChecked` type, that implements this check within it, but ultimately decided that adding an extra type won't result in a cleaner result.

Reply:
see my other comment, I think it's a good idea to do so

Reply:
Did that into a separate package because of a cyclic dependency problem.